### ISSUE TITLE
Menu styling not responsive to device size

### ISSUE THREAD

**Issue by Gibsonion** [soflyy/oxygen-bugs-and-features#434]
**Describe the bug**
When trying to edit most of the text options on lower device widths for Menu items, nothing works, all settings are locked at those set for all device widths

**Have you tried all the steps at https://oxygenbuilder.com/documentation/troubleshooting/troubleshooting-guide/?**
YES

**Are you able to replicate the issue on a Sandbox install at https://oxygenbuilder.com/try?**
http://foolish-weevil.w6.wpsandbox.pro?upass=lFq655llU6UL

**To Reproduce**
1. Create menu
2. set css for all devices
3. go to a lower device view
4. try to change any of the text option

**Expected behavior**
Something should happen

**What actually happens**
Nothing happens

**Desktop (please complete the following information):**
Chrome
Win 10
Oxy 2.4.1

**Comment by albertso** [comment]
Yes, for example if you want to change the flexbox direction from row to column on mobile you have to create a selector targeting the menu.

We should be able to change this within the menu block.

**Comment by Gibsonion** [comment]
That's actually a different problem. That is a flexbox setting being stuck on ID's and is not exclusive to menus

**Comment by lfirth** [comment]
The above still exists in Oxygen 3.1 RC 2

**Comment by Janik-Dev** [comment]
@Spellhammer the menu is not quite usable in this state. please fix soon

**Comment by davidlower8** [comment]
I can confirm this is still a bug. 

1) If I change any styling on a menu item it is reflected across all breakpoints, which it shouldn't.
2) Styling a menu item and then moving down to a smaller breakpoint, results in not being able to make changes (everything is greyed out) and styles are not applied in live mood. If I save however, the changes then exist, even though I could not see them take effect in live mood. Again if I apply changes at any breakpoint they work fine, but moving to another one, same issue, everything is greyed out and changes don't appear in live mood.

The menu item styling being reflected across all breakpoints is a pretty big bug, to be honest. Thanks for the great work Oxygen.

**Comment by shoelaced** [comment]
This, by the way, is also happening with the Social Icons block.
